yarn.lock

```diff
@@ -1690,6 +1690,16 @@
   dependencies:
     moment "^2.24.0"
 
+"@dhis2/app-runtime@^2.11.0":
```

Comment:
Let's deduplicate the lockfile with `npx yarn-deduplicate yarn.lock` as updating `app-runtime` has resulted in multiple versions of services.

Reply:
Interesting - this has been happening everywhere I've update the runtime recently

Reply:
Looks like there's a conflict with `master` now. What I usually do in this situation is:

1. Delete the `yarn.lock` in this branch and replace it with the one from `master`
2. Run `yarn install`
3. Run `npx yarn-deduplicate yarn.lock`

Reply:
Luckily there are only a few changes to yarn.lock so it's easy to fix the merge conflicts, but this seems like a useful tip for the future. Do you commit after step 1, then again after step 3? Btw after running `npx yarn-deduplicate yarn.lock` I ended up with the old 2.9.2 versions of all the runtime services still in yarn.lock, and running `yarn` again cleared those

Reply:
I do `git merge master` first (which fails because of the merge conflict), do steps 1 to 3, and then do `git commit` to commit the merge. So only one commit needed in total.

Reply:
Just tried locally and yep for some reason yarn dedup is incorrectly adding the old versions back so looks like this time there's a step 4 of `yarn install`.